packages/react/src/jsx.tsx

```diff
@@ -10,20 +11,22 @@ import {
   StateModelState,
 } from './rise'
 
-export type UI = ReactElement<Props> | ServerComponent
```

Comment:
Clean up. This is no longer needed after `ws-server` and `http-server` were removed (via models PR). For some reason, two files were left hanging, so I deleted them too!